<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="19471" started-at="2018-01-16T04:46:21Z" finished-at="2018-01-16T04:46:41Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="19471" started-at="2018-01-16T04:46:21Z" finished-at="2018-01-16T04:46:41Z">
      <class name="testCases.BookingTests">
        <test-method status="PASS" signature="prepBrowser()[pri:0, instance:testCases.BookingTests@215be6bb]" name="prepBrowser" is-config="true" duration-ms="10132" started-at="2018-01-16T15:46:21Z" description="Instantiates the browser" finished-at="2018-01-16T15:46:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepBrowser -->
        <test-method status="PASS" signature="login()[pri:0, instance:testCases.BookingTests@215be6bb]" name="login" duration-ms="3956" started-at="2018-01-16T15:46:31Z" finished-at="2018-01-16T15:46:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="enterDetails()[pri:0, instance:testCases.BookingTests@215be6bb]" name="enterDetails" duration-ms="1622" started-at="2018-01-16T15:46:35Z" finished-at="2018-01-16T15:46:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterDetails -->
        <test-method status="PASS" signature="selectFlight()[pri:0, instance:testCases.BookingTests@215be6bb]" name="selectFlight" duration-ms="2141" started-at="2018-01-16T15:46:37Z" finished-at="2018-01-16T15:46:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- selectFlight -->
        <test-method status="FAIL" signature="bookFlight()[pri:0, instance:testCases.BookingTests@215be6bb]" name="bookFlight" duration-ms="1586" started-at="2018-01-16T15:46:39Z" finished-at="2018-01-16T15:46:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at testCases.BookingTests.bookFlight(BookingTests.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bookFlight -->
      </class> <!-- testCases.BookingTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
